### PR TITLE
Add documentation for tedee integration

### PR DESCRIPTION
**Pull request by zweckj**
## Type of change

- [ ] Spelling, grammar or other readability improvements (`current` branch).
- [ ] Adjusted missing or incorrect information in the current documentation (`current` branch).
- [x] Added documentation for a new integration I'm adding to Home Assistant (`next` branch).
  - [x] I've opened up a PR to add logo's and icons in [Brands repository](https://github.com/home-assistant/brands).
- [ ] Added documentation for a new feature I'm adding to Home Assistant (`next` branch).
- [ ] Removed stale or deprecated documentation.

## Additional information

- Link to parent pull request in the codebase: https://github.com/home-assistant/core/pull/102846
- Link to parent pull request in the Brands repository: https://github.com/home-assistant/brands/pull/4822
- This PR fixes or closes issue: fixes #

## Checklist

- [x] This PR uses the correct branch, based on one of the following:
  - I made a change to the existing documentation and used the `current` branch.
  - I made a change that is related to an upcoming version of Home Assistant and used the `next` branch.
- [x] The documentation follows the Home Assistant documentation [standards].

[standards]: https://developers.home-assistant.io/docs/documenting/standards

### CODE REVIEW

File: source/_integrations/tedee.markdown [hunk 1]
```diff
@@ -0,0 +1,34 @@
+---
+title: tedee
+description: Instructions on how to integrate your Tedee lock with Home Assistant.
+ha_release: 2024.1
```
Comment (frenck) [inline source/_integrations/tedee.markdown]:
```suggestion
ha_release: 2024.2
```

File: source/_integrations/tedee.markdown [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+ha_platforms:
+  - lock
+ha_codeowners:
+  - '@zweckj'
+  - '@patrickhilker'
+ha_integration_type: integration
+---
+
+This integration interacts with your [tedee](https://tedee.com) locks, by communicating with the tedee bridge through HTTP. **The bridge is a requirement for adding your locks through this integration.**
+The integration will communicate with your lock locally, so you need to have the local API enabled.
```
Comment (frenck) [inline source/_integrations/tedee.markdown]:
```suggestion
This integration interacts with your [Tedee](https://tedee.com) locks by communicating with the Tedee bridge through HTTP. The integration will communicate with your lock locally.

## Pre-requisites

- You will need the bridge for adding your locks using this integration.
- You need to have the local API enabled.
```

File: source/_integrations/tedee.markdown [hunk 3]
```diff
@@ -0,0 +1,34 @@
...
+
+If you do not own the bridge, you can still add your locks through the [HomeKit Integration](./_integrations/homekit.markdown) to Home Assistant, but communication will happen over Bluetooth in that case and features will be limited.
```
Comment (frenck) [inline source/_integrations/tedee.markdown]:
```suggestion
If you do not own the bridge, you can still add your locks to Home Assistant through the [HomeKit Integration](./_integrations/homekit.markdown). Still, communication will happen over Bluetooth in that case, and features will be limited.
```

File: source/_integrations/tedee.markdown [hunk 4]
```diff
@@ -0,0 +1,34 @@
...
+
+{% include integrations/config_flow.md %}
+
+{% configuration_basic %}
+Host:
+  description: "The IP address of your bridge. You can find it in your router or in the tedee app under \"Bridge Settings\" -> \"Local API\"."
```
Comment (frenck) [inline source/_integrations/tedee.markdown]:
```suggestion
  description: "The IP address of your bridge. You can find it in your router or in the Tedee app under \"Bridge Settings\" -> \"Local API\"."
```

File: source/_integrations/tedee.markdown [hunk 5]
```diff
@@ -0,0 +1,34 @@
...
+  required: false
+  type: string
+Local Access Token:
+  description: "The local access token for your bridge. You can find it in the tedee app under \"Bridge Settings\" -> \"Local API\"."
```
Comment (frenck) [inline source/_integrations/tedee.markdown]:
```suggestion
  description: "The local access token for your bridge. You can find it in the Tedee app under \"Bridge Settings\" -> \"Local API\"."
```

**Review by frenck — CHANGES_REQUESTED** [review]

File: source/_integrations/tedee.markdown (hunk 2 above)
Comment (zweckj) [reply source/_integrations/tedee.markdown]: tedee as brand is all lowercase

File: source/_integrations/tedee.markdown (hunk 4 above)
Comment (zweckj) [reply source/_integrations/tedee.markdown]: Same as above

File: source/_integrations/tedee.markdown (hunk 2 above)
Comment (frenck) [reply source/_integrations/tedee.markdown]: That was my first thought too... however, that seems to be mostly logo, not textual.

From their own homepage: <https://tedee.com/>

CleanShot 2023-12-29 at 18 29 37@2x

CleanShot 2023-12-29 at 18 29 46@2x

CleanShot 2023-12-29 at 18 29 55@2x

CleanShot 2023-12-29 at 18 30 03@2x

CleanShot 2023-12-29 at 18 30 21@2x

File: source/_integrations/tedee.markdown [hunk 6]
```diff
@@ -0,0 +1,34 @@
+---
+title: tedee
```
Comment (frenck) [inline source/_integrations/tedee.markdown]:
```suggestion
title: Tedee
```

This needs to be adjusted / matched in Core too.

**Review by frenck — CHANGES_REQUESTED** [review]

File: source/_integrations/tedee.markdown (hunk 6 above)
Comment (zweckj) [reply source/_integrations/tedee.markdown]: The name should be upper case in core (like any other integration) but the manufacturer name on the device information is lower case (like in reality)

File: source/_integrations/tedee.markdown (hunk 2 above)
Comment (zweckj) [reply source/_integrations/tedee.markdown]: Huh. The more you know. I was 100% sure they wrote it lowercase.

File: source/_integrations/tedee.markdown (hunk 6 above)
Comment (zweckj) [reply source/_integrations/tedee.markdown]: matching this in core https://github.com/home-assistant/core/pull/106685

File: source/_integrations/tedee.markdown [hunk 7]
```diff
@@ -0,0 +1,41 @@
+---
+title: Tedee
+description: Instructions on how to integrate your Tedee lock with Home Assistant.
+ha_release: 2024.2
+ha_category:
+  - Lock
```
Comment (joostlek) [inline source/_integrations/tedee.markdown:7]: sensor aswell

File: source/_integrations/tedee.markdown [hunk 8]
```diff
@@ -0,0 +1,41 @@
...
+ha_iot_class: Local Push
+ha_config_flow: true
+ha_domain: tedee
+ha_platforms:
+  - lock
```
Comment (joostlek) [inline source/_integrations/tedee.markdown:14]: sensor aswell

File: source/_integrations/tedee.markdown (hunk 7 above)
Comment (zweckj) [reply source/_integrations/tedee.markdown:7]: damn forgot those, sorry

File: source/_integrations/tedee.markdown [hunk 9]
```diff
@@ -0,0 +1,48 @@
...
+  - '@zweckj'
+  - '@patrickhilker'
+ha_integration_type: integration
+---
+
+This integration interacts with your [Tedee](https://tedee.com) locks by communicating with the Tedee bridge through HTTP. The integration will communicate with your lock locally.
+
+## Pre-requisites
+
+- You will need the bridge for adding your locks using this integration.
```
Comment (frenck) [inline source/_integrations/tedee.markdown]:
```suggestion
- You will need the bridge to add your locks using this integration.
```

File: source/_integrations/tedee.markdown [hunk 10]
```diff
@@ -0,0 +1,48 @@
...
+- You need to have the local API enabled.
+
+If you do not own the bridge, you can still add your locks to Home Assistant through the [HomeKit Integration](./_integrations/homekit.markdown). Still, communication will happen over Bluetooth in that case, and features will be limited.
```
Comment (frenck) [inline source/_integrations/tedee.markdown]:
```suggestion
If you do not own the bridge, you can still add your locks to Home Assistant through the [HomeKit device integration](/integrations/homekit_controller.markdown). Communication will happen over Bluetooth in that case, and features will be limited.
```

File: source/_integrations/tedee.markdown [hunk 11]
```diff
@@ -0,0 +1,48 @@
...
+
+{% include integrations/config_flow.md %}
+
+{% configuration_basic %}
+Host:
+  description: "The IP address of your bridge. You can find it in your router or in the Tedee app under \"Bridge Settings\" -> \"Local API\"."
```
Comment (frenck) [inline source/_integrations/tedee.markdown]:
```suggestion
  description: "The IP address of your bridge. You can find it in your router or in the Tedee app under **Bridge Settings** -> **Local API**.
```

File: source/_integrations/tedee.markdown [hunk 12]
```diff
@@ -0,0 +1,48 @@
...
+  required: false
+  type: string
+Local Access Token:
+  description: "The local access token for your bridge. You can find it in the Tedee app under \"Bridge Settings\" -> \"Local API\"."
```
Comment (frenck) [inline source/_integrations/tedee.markdown]:
```suggestion
  description: "The local access token for your bridge. You can find it in the Tedee app under **Bridge Settings** -> **Local API**."
```

File: source/_integrations/tedee.markdown [hunk 13]
```diff
@@ -0,0 +1,48 @@
...
+  required: false
+  type: string
+{% endconfiguration_basic %}
+
+## Binary sensors
+We have three binary sensors: One that indicated whether the battery is currently charging, one indicating if pullspring is enabled and one indicating whether the lock is in a "semi locked" position, meaning the lock has been turned manually and is between its normal end positions.
```
Comment (frenck) [inline source/_integrations/tedee.markdown]:
```suggestion
## Binary sensors

We have three binary sensors: One that indicates whether the battery is currently charging, one indicating if the pull spring is enabled, and one indicating whether the lock is in a "semi-locked" position, meaning the lock has been turned manually and between its normal end positions.
```

File: source/_integrations/tedee.markdown [hunk 14]
```diff
@@ -0,0 +1,48 @@
...
+
+## Sensors
+The integration currently offers two sensors: A battery sensor, indicated the charge of your lock and a "pullspring duration" sensor, indicating how long (in seconds) your latch will stay pulled after a pull operation (if supported).
```
Comment (frenck) [inline source/_integrations/tedee.markdown]:
```suggestion
## Sensors

The integration currently offers two sensors: A battery sensor, indicating the charge of your lock, and a "pull spring duration" sensor, indicating how long (in seconds) your latch will stay pulled after a pull operation (if supported). 
```

**Review by frenck — CHANGES_REQUESTED** [review]

**Review by frenck — DISMISSED** [review]
Thanks, @zweckj 👍 

../Frenck

File: source/_integrations/tedee.markdown [hunk 15]
```diff
@@ -0,0 +1,50 @@
...
+- You will need the bridge to add your locks using this integration.
+- You need to have the local API enabled.
+
+If you do not own the bridge, you can still add your locks to Home Assistant through the [HomeKit device integration](/integrations/homekit_controller.markdown). Communication will happen over Bluetooth in that case, and features will be limited.
+
+{% include integrations/config_flow.md %}
+
+{% configuration_basic %}
+Host:
+  description: "The IP address of your bridge. You can find it in your router or in the Tedee app under **Bridge Settings** -> **Local API**.
```
Comment (frenck) [inline source/_integrations/tedee.markdown]:
```suggestion
  description: "The IP address of your bridge. You can find it in your router or in the Tedee app under **Bridge Settings** -> **Local API**."
```

**Review by frenck — APPROVED** [review]

**Pull request merged**